lib/src/acrcloud_response.dart

```diff
@@ -32,9 +32,12 @@ class ACRCloudResponseStatus {
 /// The metadata of the response, which contains all of the matching tracks.
 @JsonSerializable()
 class ACRCloudResponseMetadata {
-  final List<ACRCloudResponseMusicItem> music;
+  final List<ACRCloudResponseMusicItem>? music;
```

Comment:
What do you think about giving these lists `defaultValue`s of `const <ACRCloudResponseMusicItem>[]` and `const <>[ACRCloudResponseCustomFileItem]`, respectively? Then, they won't be nullable and you can just do an `isEmpty` check if you want.

https://pub.dev/documentation/json_annotation/latest/json_annotation/JsonKey/defaultValue.html

Reply:
Thanks for this! I was actually looking for a way to do this, didn't know json_annotation already had one.